standard.md

```diff
@@ -6,17 +6,17 @@
+**2. Use of Approved Open Licenses** | Digital public goods must demonstrate the use of an approved open license. For open-source software, only [OSI approved licenses](https://opensource.org/licenses) are accepted. For open content collections the use of a [Creative Commons license](https://creativecommons.org/licenses/) is required. DPGs are encourageed to use a license that allows for both derivatives and commercial reuse ([CC-BY](https://creativecommons.org/licenses/by/4.0/) and [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)), or dedicate content to the public domain ([CC0](https://creativecommons.org/choose/zero/)); licenses that do not allow for commercial reuse ([CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/)) are also accepted. For open data, an [Open Data Commons approved license](https://opendefinition.org/licenses/) is required. See [The full license list](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md) for reference.
+**3. Clear Ownership** | Ownership of assets that the digital public good produces must be clearly defined and documented. For example, through copyright, trademark or other publicly available information.
+**4. Platform Independence** | When the digital public good has mandatory dependencies that create more restrictions than the original license, proving independence from the closed component(s) and/or indicating the existence of functional, open alternatives that can be used without significant changes to the core product is required.
+**5. Documentation** | Digital public goods require documentation of the source code, use cases, and/or functional requirements. For content collections, this should include all relevant/compatible apps, software, or hardware required to access the content collection, and instructions regarding how to use it. For software solutions, this should be technical documentation that would allow a technical person unfamiliar with the project to launch and run the software. For data projects, this should be documentation that describes all the fields in the set, and provides context on how the dataset was collected, and how it should be interpreted.
+**6. Mechanism for Extracting Data and Content** | Digital Public Goods with non personally identifiable information (PII) design for possibility of extracting or importing non-PII data and content from the system in a non-proprietary format.
+**7. Adherence to Privacy and Applicable Laws** | Digital Public Goods must be designed and developed to comply with applicable privacy laws.
+**8. Adherence to Standards & Best Practices** | Digital Public Goods must be designed and developed to align with relevant standards, best practices, and/or principles. For example,[the Principles for Digital Development](https://digitalprinciples.org/principles/).
```

Comment:
Some standards and best practices are written with respect to product design and would be applicable to the DPGs themselves, while some others (including the Digital Principles) **also** include recommendations for the people and organizations producing those products -- it might be good to write this in a way that highlights both aspects.

Reply:
Fair point. However, we do not want to be prescriptive about models of governance. Like the point above, our main concern is potential to do harm.